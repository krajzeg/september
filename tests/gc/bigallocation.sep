string := "a"
while (string.length() < 1000000) {
	string = string + string
}

print("Allocated a string of", string.length(), "characters successfully.")