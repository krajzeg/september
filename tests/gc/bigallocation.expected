Allocated a string of 1048576 characters successfully.
